ketcher-autotests/playwright.config.ts

```diff
@@ -0,0 +1,126 @@
+import * as dotenv from 'dotenv';
```

Comment:
@AnastasiiaPlyako perhaps we should double-check if this file was updated in the current autotests repository

Reply:
done